__tests__/app.test.js

```diff
@@ -199,4 +294,21 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.message).toBe("Not found");
         });
     });
+});
+
+describe('GET /api/users', () => {
+    it('returns status 200 and returns all users, with properties of username, name, avatar_url ', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body}) => {
+            const firstUser = body.users[0];
```

Comment:
This would be better if you used a forEach to check each user's properties, as you have done with your other tests.